### DISCUSSION TITLE
CsvWriter does not respect CultureInfo

### DISCUSSION THREAD

**Discussion by StanDeMan** [JoshClose/CsvHelper#1513] [Q&A]
Hi there,

thanks a lot for your amazing CsvHelper. It's pretty easy to implement.
I'm using .NET Core 3.1 and the CultureInfo seems not to work? I did this to give it a try:

```cs
records = new List<object>
{
    new { Id = 1, Name = "oneÄäÖöÜü" },
};
using (var writer = new StreamWriter(@$"{sourcePath}\test.csv"))
{
    using var csv = new CsvWriter(writer, CultureInfo.GetCultureInfo("de-DE"));
    csv.WriteRecords(records);
}
```

This is what I get in the csv file for name: oneÃ„Ã¤Ã–Ã¶ÃœÃ¼

Here is my code I'm using so far:

```cs
using (var writer = new StreamWriter(@$"{sourcePath}\file.csv"))
{
   using var csv = new CsvWriter(writer, CultureInfo.CurrentCulture = new CultureInfo("de-DE"));
   csv.WriteRecords(records);
}
```

The same result for so called German Umlaute (Ä,ä,Ö,ö,Ü,ü, etc.)?

Thanks a lot for your help.
Kindest regards

**Comment by StanDeMan** [comment]
This seems to be an Excel issue! If I take a look inside the csv file everything is okay. If I open the csv file thru excel I get this strange characters. Hm? I'm using Office 365 and there is no option for importing a CSV with UTF-8. But, this is an office problem. You can close this issue, please. Thanks.

**Comment by JoshClose** [comment]
You might need to write the file using a specific encoding, which you would set on the `StreamWriter`.

**Comment by Ekkeir** [comment]
I would agree on an encoding.
In our case something like this helped
```csharp
using (var writer = new StreamWriter(stream, Encoding.UTF8))
{...}
```
